### PR TITLE
Walking skeleton

### PR DESCRIPTION
**Pull request by jeremyschuurmans**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.vscode
```
Comment (mikeknep) [inline .gitignore]: This is fine, but you may want to consider adding `.vscode` to your [global gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
 # Tic-Tac-Toe
+
+[![Build status](https://travis-ci.org/palledorous/Tic-Tac-Toe.svg?master)](https://travis-ci.org/palledorous)
+
+## Usage
+
+1. To start the game, run `<insert command here>`
```
Comment (pedmiston) [inline README.md]: Let's figure out what you want the entrypoint for the game to be, and add it here, and as a file (or more) in the repo.

File: tests/smoke_test.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import pytest
```
Comment (pedmiston) [inline tests/smoke_test.py]: The import of pytest here is not strictly needed.

File: Pipfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+pylint = "*"
```
Comment (mikeknep) [inline Pipfile:8]: You might want to pin these to specific versions—tends to make updating dependencies a little more controlled (at least that's been my experience especially in Ruby with bundler; pip might have a better approach to this)

Comment (pedmiston) [inline Pipfile:8]: Another dev package that VSCode likes is "black", which is a great autoformatter for python code. You can set it up with VSCode to format your code properly whenever you save the file. ⭐️

**Review by mikeknep — APPROVED** [review]
Just a couple minor things, no blockers.

File: Pipfile (hunk 4 above)
Comment (mikeknep) [reply Pipfile:8]: Black is pretty sweet. Autoformat-on-save is even sweeter.

File: tests/smoke_test.py (hunk 3 above)
Comment (jeremyschuurmans) [reply tests/smoke_test.py]: Ah good to know! Thanks!

File: Pipfile (hunk 4 above)
Comment (jeremyschuurmans) [reply Pipfile:8]: I do want the sweetest autoformatting I can get, but I couldn't find autoformat-on-save. I'm going to go ahead and install black for this.

Comment (jeremyschuurmans) [reply Pipfile:8]: I tried to do this, but it kept getting errors. I think I'm probably not understanding the syntax or something.

**Review by pedmiston — APPROVED** [review]

**Pull request merged**